src/Symfony/Component/Cache/Traits/AbstractTrait.php

```diff
@@ -189,11 +204,18 @@ private function getId($key)
     {
         CacheItem::validateKey($key);
 
+        if ($this->versioningIsEnabled && '' === $this->namespaceVersion) {
```

Comment:
Other classes used the trick `isset($string[0])` to check if their is a namespace.